frontend/src/index.tsx

```diff
@@ -36,7 +36,7 @@ const authLink = setContext(async (_, { headers }) => {
   >(AUTHENTICATED_USER_KEY, "accessToken");
 
   if (token) {
-    const decodedToken = jwt.decode(token) as DecodedJWT;
+    const decodedToken = "" as DecodedJWT;
```

Comment:
Can you add a `TODO` comment here to note that the JWT token needs to be decoded here